content/en/docs/Reference/declarative-config.md

```diff
@@ -0,0 +1,636 @@
+    version: string & !=""
+    kind: string & !=""
+  }
+}
+```
+
+#### `olm.bundle.object` (alpha)
+
+`olm.bundle.object` properties are used to inline (or reference) a bundle's manifests directly in the index.
+
+> **NOTE**: Core OLM does not require `olm.bundle.object` properties to be included on bundles. However, the OLM Package Server (used by tooling such as the kubectl operator plugin and the OpenShift console) does require these properties to be able to serve metadata about the packages in an index.
+> 
+> This property is in _alpha_ because it will likely be rendered obsolete when updates can be made to the OLM Package Server to no longer require manifests in the index.
+
+A bundle object property can contain inlined data using the `value.data` field, which must the base64-encoded string of that manifest
+
+Alternately, a bundle object property can be a reference to a file relative to the location of file in which the bundle is declared. Any referenced files must be within the declarative config root.
+
+The `olm.bundle.object` property [cue][cuelang-spec] schema is:
+```cue
+
+#PropertyBundleObject: {
+  type: "olm.bundle.object"
+  value: #propertyBundleObjectRef | #propertyBundleObjectData
+} 
+
+#propertyBundleObjectRef: {
+    ref: string & !=""
+}
+
+#propertyBundleObjectData: {
+    data: string & !=""
+}
+```
+
+[cuelang-spec]: https://cuelang.org/docs/references/spec/
+[semver]: https://semver.org/spec/v2.0.0.html
+[semver-range]: https://github.com/blang/semver/blob/master/README.md#ranges
+
+## CLI
+
+### `opm init`
+
+```
+Generate an olm.package declarative config blob
+
+Usage:
+  opm init <packageName> [flags]
+
+Flags:
+  -c, --default-channel string   The channel that subscriptions will default to if unspecified
+  -d, --description string       Path to the operator's README.md (or other documentation)
+  -h, --help                     help for init
+  -i, --icon string              Path to package's icon
+  -o, --output string            Output format (json|yaml) (default "json")
+
+Global Flags:
+      --skip-tls   skip TLS certificate verification for container image registries while pulling bundles or index
+```
+
+### `opm render`
+
+```
+Generate a declarative config blob from the provided index images, bundle images, and sqlite database files
+
+Usage:
+  opm render [index-image | bundle-image | sqlite-file]... [flags]
+
+Flags:
+  -h, --help            help for render
+  -o, --output string   Output format (json|yaml) (default "json")
+
+Global Flags:
+      --skip-tls   skip TLS certificate verification for container image registries while pulling bundles or index
+```
+
+### `opm validate`
+
+```
+Validate the declarative config JSON file(s) in a given directory
+
+Usage:
+  opm validate <directory> [flags]
+
+Flags:
+  -h, --help   help for validate
+
+Global Flags:
+      --skip-tls   skip TLS certificate verification for container image registries while pulling bundles or index
+```
+
+### `opm serve`
```

Comment:
We possibly want to mention that changes in dc won't be detected once it starts serving.

Reply:
I'll submit a PR to the CLI help text that mentions this, and then update this accordingly.

Reply:
See operator-framework/operator-registry#750